Sources/Fluent/Entity/Entity.swift

```diff
@@ -19,6 +19,15 @@ public protocol Entity: Preparation, NodeConvertible {
         like pivots.
     */
     static var name: String { get }
+    
+    /**
+        The name of the column that corresponds
+        to this entity's key.
+     
+        Return nil to use the database's global
+        idKey.
+     */
+    static var idKey: String? { get }
```

Comment:
In the beginning I had a rationale for that choice, but forgot to note it down, and I guess it became invalid during implementation, anyways. I'll switch it to a default and then we'll see how it looks.